cost-analyzer/templates/_helpers.tpl

```diff
@@ -1683,4 +1647,22 @@ for more information
 {{- $parts := splitList ":" $image }}
 {{- $tag := last $parts }}
 {{- $tag }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+
+{{- define "finops-agent.clusterId" -}}
+{{- if .Values.global.clusterId -}}
+{{ .Values.global.clusterId }}
+{{- else -}}
+{{ include "mustHaveGlobalFederatedStoreCheck" . }}
+{{ fail "\n\nKubecost 2.9.x is intended as a prerequisite to upgrade to 3.0.\nglobal.clusterId is required. Please set .Values.global.clusterId. This key replaces the previous key used: .Values.prometheus.server.global.external_labels.cluster_id\nFor more information, see: TODO: link to 2.9 examples" }}
+{{- end -}}
+{{- end -}}
```

Comment:
Is this function called anywhere? We should make sure it is called, to validate users have set a `clusterId`

Reply:
yes, it is in NOTES also.